config/vufind/ModelContextProtocol.yaml

```diff
@@ -0,0 +1,74 @@
+# Model Context Protocol (MCP) server configuration.
+#
+# See (a wiki page -- I don't have wiki permissions to create a page)
```

Comment:
@demiankatz please create a page at https://vufind.org/wiki/configuration:model_context_protocol (or give me that permission)

Reply:
And I've drafted the page and linked it from the config.